openmetadata-spec/src/main/resources/json/schema/entity/services/connections/database/databricksConnection.json

```diff
@@ -48,6 +48,11 @@
       "description": "Databricks compute resources URL.",
       "type": "string"
     },
+    "catalog": {
```

Comment:
are we using this property anywhere?

Reply:
Yes, that will be used when we want to ingest a specific catalog (Otherwise it will ingest all the catalog from databricks).

Reply:
cool got it

Reply:
hi @MilanBariya could you please add some more details around here. If I understand correctly, passing the `catalog` will be used as a way to ingest a single "database" in OM. If it is left null, we'll iterate over all catalogs. Let's add all this information here to help users understand how this param is used.

Thanks